src/routes/cohort.js

```diff
@@ -10,5 +16,16 @@ const router = Router()
 router.post('/', validateAuthentication, validateTeacherRole, create)
 router.get('/', validateAuthentication, validateTeacherRole, cohorts)
 router.get('/:id', validateAuthentication, validateTeacherRole, getCohortById)
+router.get(
+  '/:id/cohortExercises',
+  validateAuthentication,
+  getAllCohortExercises
+)
+router.post(
+  '/:id/cohortExercise',
```

Comment:
update path to use plural resource: `'/:id/cohortExercises'`

Reply:
done